tests/modules/test_patch.py

```diff
@@ -94,7 +94,7 @@ def test_create_patch_change(self):
 
         patch_fn = f"{'-'.join(BISMARK_ALIGN.split('/'))}.diff"
         # Check that a patch file with the correct name has been created
-        assert set(os.listdir(module_path)) == {"main.nf", "meta.yml", "environment.yml", patch_fn}
+        assert patch_fn in set(os.listdir(module_path))
```

Comment:
how about we test directly the file existence:
```suggestion
       assert Path(module_path) / patch_fn.exists()
```
same for the others

Reply:
makes sense, thanks for the suggestion 😄